src/styles/utils/variables.scss

```diff
@@ -0,0 +1,6 @@
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+$gray-elements: #f3f3f3;
+$star-size: 16px;
+$buy-button-height: 40px;
```

Comment:
Don't use names of colors in variable name(for example 'blue', 'gray'....)
Better to name it like `main-color`, `button-hover-color`....